src/Uno.Extensions.Configuration/ConfigurationBuilderExtensions.cs

```diff
@@ -5,7 +5,11 @@ public static class ConfigurationBuilderExtensions
 	private static IConfigurationBuilder AddConfigurationFile(this IConfigurationBuilder configurationBuilder, HostBuilderContext hostingContext, string configurationFileName)
 	{
 		var relativePath = $"{ConfigBuilderExtensions.ConfigurationFolderName}/{configurationFileName}";
-		var rootFolder = (hostingContext.HostingEnvironment as IAppHostEnvironment)?.AppDataPath ?? String.Empty;
+		var rootFolder = (hostingContext.HostingEnvironment as IAppHostEnvironment)?.AppDataPath;
```

Comment:
both here and `src/Uno.Extensions.Configuration/ConfigBuilderExtensions.cs` you have this same line of code... would suggest maybe moving this to an extension method? Do we maybe want to pass the Logger in and have something more like:
```cs
internal static string? GetAppDataPath(this IAmHctxHostingEnvironment env, ILogger logger)
{
    if (hctx.HostingEnvironment is IAppHostEnvornment appHostEnv) 
    { 
        return appHostEnv.AppDataPath; 
    } 
    else if (logger.IsEnabled(LogLevel.Warning)) 
    { 
        // seems like this shouldn't happen but do we want to generate a warning if it does?
        logger.LogWarningMessage("....");
    } 

    return default;
}
```